pulsar-function-go/pf/instanceConf.go

```diff
@@ -24,7 +24,7 @@ import (
 	"time"
 
 	"github.com/apache/pulsar/pulsar-function-go/conf"
-	"github.com/apache/pulsar/pulsar-function-go/pb"
+	pb "github.com/apache/pulsar/pulsar-function-go/pb"
```

Comment:
```suggestion
	"github.com/apache/pulsar/pulsar-function-go/pb"
```

Reply:
Again, when I make this change, I can't get it to build.